src/main/java/org/alliancegenome/curation_api/services/LoggedInPersonService.java

```diff
@@ -21,6 +23,11 @@ protected void init() {
         setSQLDao(loggedInPersonDAO);
     }
     
+    public void saveSettings(HashMap<String, Object> settings) {
```

Comment:
@markquintontulloch this method will need a `@Transactional` and you can persist vs merge for the save.

Reply:
If fact you probably don't even need the persist or merge considering its inside the transactional any changes will just happen and get persisted back to the db.

Reply:
Done